### ISSUE TITLE
Run backburner in production mode

### ISSUE THREAD

**Issue by Ahmed--Mohsen** [nesquena/backburner#65]
When I ran the following command backburn ran in development mode instead of production 
`bundle exec backburner -q newsletter-sender,push-notifier -d -P /var/run/backburner.pid -l /var/log/backburner.log RAILS_ENV=production`
How to specify the environment?

**Comment by contentfree** [comment]
Once #101 is merged in, you will be able to provide `-e production`

**Comment by contentfree** [comment]
@vad4msiu's suggestion would've worked. Now that #101 has been merged into master, you are also able to use the `-e` switch to control the environment (for Rails and Padrino apps) for the CLI. @Ahmed--Mohsen: Please confirm and close the issue.

**Comment by Ahmed--Mohsen** [comment]
Confirmed
